src/pkey/clu_pkey.c

```diff
@@ -141,6 +142,78 @@ static int wolfCLU_pKeyPEMtoPubKey(WOLFSSL_BIO* bio, WOLFSSL_EVP_PKEY* pkey)
 }
 
 
+/* return WOLFCLU_SUCCESS on success */
+static int wolfCLU_DerToEncryptedPEM(WOLFSSL_BIO* bio, byte* key, word32 keySz,
+        int encAlgId, byte* password, word32 passwordSz)
+{
+    int ret = WOLFCLU_SUCCESS;
+    byte* out    = NULL;
+    word32 outSz = 0;
+    WC_RNG rng;
+
+    byte* pemBuf  = NULL;
+    int   pemBufSz;
+    byte* salt    = NULL;
+    word32 saltSz = 0;
+    int itt       = 1000;
```

Comment:
Is this number of PBKDF iterations an enum/macro in wolfSSL somewhere for PKCS8? If not let's make note to add one. I'm guessing the 1000 is a standard number for PKCS8. Something like we do for `WC_PKCS12_ITT_DEFAULT`. 

Something like this:
```
#ifndef WC_PKCS8_ITT_DEFAULT
#define WC_PKCS8_ITT_DEFAULT 1000
#endif
```
And add to `#define WC_PKCS8_ITT_DEFAULT 1000` to asn_public.h.

Reply:
Thanks David, for now will set it to the PKCS12 default. Have itt and salt variable because I forsee this code being reused if PKCS8 support is added or PKCS12 creation.